ResearchKit/Common/ORKImageCaptureStepViewController.m

```diff
@@ -315,7 +315,7 @@ - (void)setCapturedImageData:(NSData *)capturedImageData {
 }
 
 - (NSURL *)writeCapturedDataWithError:(NSError **)error {
-    NSURL *URL = [self.outputDirectory URLByAppendingPathComponent:[NSString stringWithFormat:@"%@.jpg",self.step.identifier]];
+    NSURL *URL = [self.outputDirectory URLByAppendingPathComponent:[NSString stringWithFormat:@"%@.jpg", [[NSUUID UUID] UUIDString]]];
```

Comment:
How are these images gonna be fetched once they're saved.

Reply:
While task is completed locally,  `fileURL` property of `ORKFileResult` is set to that URL which is unique for each captured image (if image is recaptured, file at previously assigned path is deleted, and new URL is generated).

When image is remotely fetched, it's stored at:
`let outputDirectoryURL = MDRAxon.generalOutputDirectory().appendingPathComponent(id.stringRepresentation())`

which is assigned to `ORKFileResult` -> `fileURL` property (`MDRImageCaptureStep+MedableAxon.swift`